Sports/live_nba.1m.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env PYTHONIOENCODING=UTF-8 /usr/local/bin/python3
+
+# <bitbar.title>Live NBA Game Stat</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Chen Luxin</bitbar.author>
+# <bitbar.author.github>chenlujjj</bitbar.author.github>
+# <bitbar.desc>Show NBA game scores in bar.</bitbar.desc>
+# <bitbar.image>https://user-images.githubusercontent.com/17081528/101980272-3b763000-3c9f-11eb-9f0e-a14163725805.png</bitbar.image>
+# <bitbar.dependencies>python3</bitbar.dependencies>
+
+import json
+import urllib.request
+from datetime import datetime, timezone
+
+
+def time_convert(dt):
+    local_datetime = dt.replace(tzinfo=timezone.utc).astimezone(tz=None)
+    # only care about time, not date
+    return datetime.strftime(local_datetime, '%H:%M')
+
+
+class Game:
+
+    @classmethod
+    def from_data(cls, data: dict):
+        game = cls()
+        fmt = '%Y-%m-%dT%H:%M:%S.%f%z'
+        game.start_time = datetime.strptime(data['startTimeUTC'], fmt)
+        if 'endTimeUTC' in data:
+            game.end_time = datetime.strptime(data['endTimeUTC'], fmt)
+        else:
+            game.end_time = None
+        game.hTeam = Team.from_data(data['hTeam'])
+        game.vTeam = Team.from_data(data['vTeam'])
+        return game
+
+    def __repr__(self):
+        # example: 09:00-11:35  MIA 98-116 LAL
+        start_time = time_convert(self.start_time)
+        end_time = time_convert(self.end_time) if self.end_time else 'now'
+        return f'{start_time}-{end_time} {self.vTeam.triCode} {self.vTeam.score}-{self.hTeam.score} {self.hTeam.triCode}'
+
+    def details(self):
+        vTeam_linescores = self.vTeam.linescores
+        hTeam_linescores = self.hTeam.linescores
+        return " ".join(["%s:%s" % (vTeam_linescores[i], hTeam_linescores[i]) for i in range(len(vTeam_linescores))])
+
+
+class Team:
+    def __init__(self, triCode: str, score: int, linescore):
+        self.triCode = triCode
+        self.score = score
+        self.linescore = linescore
+
+    @classmethod
+    def from_data(cls, data: dict):
+        team = cls(data['triCode'], int(data['score']), data['linescore'])
```

Comment:
This currently fails before the game starts because data['score'] is an empty string -- try this:
```suggestion
        team = cls(data['triCode'], int(data['score'] or 0), data['linescore'])
```